src/main/java/apoc/util/s3util/S3Params.java

```diff
@@ -0,0 +1,45 @@
+package apoc.util.s3util;
+
+public class S3Params {
+
+    private final String accessKey;
+    private final String secretKey;
```

Comment:
good question, we should have an option to store this in a separate config, either in neo4j.conf or in a more secure place.